test/integration/drivers/amazonec2/create-ebsinstance.bats

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bats
+
+load ${BASE_TEST_DIR}/helpers.bash
+
+force_env DRIVER amazonec2
+
+#Use Instance Type that supports EBS Optimize 
+export AWS_INSTANCE_TYPE=m4.large
+
+require_env AWS_DEFAULT_REGION
```

Comment:
It is required by the aws cli.  Otherwise it will complain there is no default region.  

> On Oct 7, 2015, at 11:17 PM, Nathan LeClaire <notifications@github.com> wrote:
> 
> In test/integration/drivers/amazonec2/create-ebsinstance.bats:
> 
> > @@ -0,0 +1,30 @@
> > +#!/usr/bin/env bats
> > +
> > +load ${BASE_TEST_DIR}/helpers.bash
> > +
> > +force_env DRIVER amazonec2
> > +
> > +#Use Instance Type that supports EBS Optimize 
> > +export AWS_INSTANCE_TYPE=m4.large
> > +
> > +require_env AWS_DEFAULT_REGION
> Why is this required? It should just default to us-east-1, right?
> 
> —
> Reply to this email directly or view it on GitHub.
>